requirements.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Autodesk, Inc.  All rights reserved.
+#
+# Use of this software is subject to the terms of the Autodesk license agreement
+# provided at the time of installation or download, or which otherwise accompanies
+# this software in either electronic or hard copy form.
+#
+
+sphinx==1.8.3
```

Comment:
At some point we should have a discussion about `pipenv`. It's basically pip + a lock file. The problem with using `pip` as is is that while your direct dependencies are locked, your indirect dependencies are not, which means rebuilding the same commit over time can generate different results.